### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by aknysh**
## what
* Initial implementation of Terraform module for a DocumentDB cluster

## why
* Provision a DocumentDB cluster on AWS

## test

image

image

### CODE REVIEW

File: README.yaml [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  - name: db-instance-class-specs
+    description: Amazon DocumentDB Instance Class Specifications
+    url: https://docs.aws.amazon.com/documentdb/latest/developerguide/db-instance-classes.html#db-instance-class-specs
+  - name: db-cluster-parameter-group-create
+    description: Creating an Amazon DocumentDB Cluster Parameter Group
+    url: https://docs.aws.amazon.com/documentdb/latest/developerguide/db-cluster-parameter-group-create.html
+  - name: limits
+    description: Amazon DocumentDB Limits
+    url: https://docs.aws.amazon.com/documentdb/latest/developerguide/limits.html
+  - name: documentdb-arns
```
Comment (osterman) [inline README.yaml]: we use a programmatic name for repos, but I think for articles it makes more sense to use a title.

Comment (aknysh) [reply README.yaml]: you mean the same as `description`?
description: Amazon DocumentDB Limits

**Comment by osterman** [comment]
Build failing

**Comment by aknysh** [comment]
no, Travis can't build the master branch b/c it's empty

**Comment by aknysh** [comment]
Travis started building other branches, e.g. `init` 
https://travis-ci.org/cloudposse/terraform-aws-documentdb-cluster/builds/514761674

File: .gitignore [hunk 2]
```diff
@@ -1,9 +1,13 @@
-#  Local .terraform directories
-**/.terraform/*
-
-# .tfstate files
+# Compiled files
 *.tfstate
-*.tfstate.*
```
Comment (Nuru) [inline .gitignore]: What's wrong with `*.tfstate.*`?

File: README.yaml (hunk 1 above)
Comment (Nuru) [reply README.yaml]: I agree with osterman. I think you should use what you have as descriptions for the titles, and then the descriptions can be longer, like "Official AWS documentation on how to create a DocumentDB cluster" or just a repeat of the title but without title case.

File: .gitignore (hunk 2 above)
Comment (aknysh) [reply .gitignore]: nothing wrong with it.
the file just gets copied from one repo or another, there they have different implementations (we need to standardize that)

**Pull request merged**